<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Basketball</class>
 <widget class="QWidget" name="Basketball">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>682</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="page1">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="Background1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1024</width>
       <height>682</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/background2.PNG);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ProjectName">
     <property name="geometry">
      <rect>
       <x>237</x>
       <y>110</y>
       <width>550</width>
       <height>140</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 75pt &quot;Bauhaus 93&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Right Now!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AnalyseMode">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>270</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(120, 170, 195);
color: rgb(255, 255, 255);
font: 15pt &quot;Bauhaus 93&quot;;</string>
     </property>
     <property name="text">
      <string>Analyse</string>
     </property>
    </widget>
    <widget class="QPushButton" name="TrainMode">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>270</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(120, 170, 195);
color: rgb(255, 255, 255);
font: 15pt &quot;Bauhaus 93&quot;;</string>
     </property>
     <property name="text">
      <string>Train</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page2">
    <widget class="QLabel" name="AnalyseVideo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1024</width>
       <height>682</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Back1">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>640</y>
       <width>81</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0,0,0);
font: 15pt &quot;Bauhaus 93&quot;;</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="Hit">
     <property name="geometry">
      <rect>
       <x>412</x>
       <y>570</y>
       <width>200</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 64);
font: 48pt &quot;Bauhaus 93&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;1 / 10&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page3">
    <widget class="QLabel" name="Total1">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>240</y>
       <width>220</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 40pt &quot;Bauhaus 93&quot;;</string>
     </property>
     <property name="text">
      <string>Total:</string>
     </property>
    </widget>
    <widget class="QLabel" name="Hit2">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>170</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 40pt &quot;Bauhaus 93&quot;;</string>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Back2">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>640</y>
       <width>81</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0,0,0);
font: 15pt &quot;Bauhaus 93&quot;;</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="Background3">
     <property name="geometry">
      <rect>
       <x>332</x>
       <y>146</y>
       <width>360</width>
       <height>390</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 65);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Hit1">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>170</y>
       <width>220</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 40pt &quot;Bauhaus 93&quot;;</string>
     </property>
     <property name="text">
      <string>Hit:</string>
     </property>
    </widget>
    <widget class="QLabel" name="Rate1">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>310</y>
       <width>220</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 40pt &quot;Bauhaus 93&quot;;</string>
     </property>
     <property name="text">
      <string>Hit Rate:</string>
     </property>
    </widget>
    <widget class="QLabel" name="Speed1">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>380</y>
       <width>220</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 40pt &quot;Bauhaus 93&quot;;</string>
     </property>
     <property name="text">
      <string>Speed:</string>
     </property>
    </widget>
    <widget class="QLabel" name="Angle1">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>450</y>
       <width>220</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 40pt &quot;Bauhaus 93&quot;;</string>
     </property>
     <property name="text">
      <string>Angle:</string>
     </property>
    </widget>
    <widget class="QLabel" name="Total2">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>240</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 40pt &quot;Bauhaus 93&quot;;</string>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLabel" name="Rate2">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>310</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 40pt &quot;Bauhaus 93&quot;;</string>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLabel" name="Speed2">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>380</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 40pt &quot;Bauhaus 93&quot;;</string>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLabel" name="Angle2">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>450</y>
       <width>100</width>
       <height>60</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 40pt &quot;Bauhaus 93&quot;;</string>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <zorder>Background3</zorder>
    <zorder>Total1</zorder>
    <zorder>Hit2</zorder>
    <zorder>Back2</zorder>
    <zorder>Hit1</zorder>
    <zorder>Rate1</zorder>
    <zorder>Speed1</zorder>
    <zorder>Angle1</zorder>
    <zorder>Total2</zorder>
    <zorder>Rate2</zorder>
    <zorder>Speed2</zorder>
    <zorder>Angle2</zorder>
   </widget>
   <widget class="QWidget" name="page4">
    <widget class="QLabel" name="Background4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1024</width>
       <height>682</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/background3.PNG);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="BasketballMode">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>261</y>
       <width>241</width>
       <height>160</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 35pt &quot;Bauhaus 93&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Basketball &lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Mode&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="BasketballIcon">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>291</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/basketball2.PNG);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="HandIcon">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>291</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/hand3.PNG);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="HandMode">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>260</y>
       <width>221</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 35pt &quot;Bauhaus 93&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Hand&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Mode&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page5">
    <widget class="QLabel" name="Video5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1024</width>
       <height>682</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Points1">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>550</y>
       <width>190</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255,255,255);
font: 48pt &quot;Bauhaus 93&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;10&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="BasketballIcon1">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>240</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/basketball.PNG);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Back5">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>640</y>
       <width>81</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0,0,0);
font: 15pt &quot;Bauhaus 93&quot;;</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="Time1">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>10</y>
       <width>185</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255,255,255);
font: 50pt &quot;Bauhaus 93&quot;;</string>
     </property>
     <property name="text">
      <string>01:00</string>
     </property>
    </widget>
    <widget class="QLabel" name="TimeBackground1">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>10</y>
       <width>185</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 64);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="PointsBackground1">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>550</y>
       <width>190</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 64);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>TimeBackground1</zorder>
    <zorder>PointsBackground1</zorder>
    <zorder>Video5</zorder>
    <zorder>Points1</zorder>
    <zorder>BasketballIcon1</zorder>
    <zorder>Back5</zorder>
    <zorder>Time1</zorder>
   </widget>
   <widget class="QWidget" name="page6">
    <widget class="QLabel" name="Video6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1024</width>
       <height>682</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="HandIcon1">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>260</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/hand.PNG);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Time2">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>10</y>
       <width>185</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255,255,255);
font: 50pt &quot;Bauhaus 93&quot;;</string>
     </property>
     <property name="text">
      <string>01:00</string>
     </property>
    </widget>
    <widget class="QLabel" name="TimeBackground2">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>10</y>
       <width>185</width>
       <height>90</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 64);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Back6">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>640</y>
       <width>81</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0,0,0);
font: 15pt &quot;Bauhaus 93&quot;;</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QLabel" name="Points2">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>550</y>
       <width>190</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255,255,255);
font: 48pt &quot;Bauhaus 93&quot;;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;10&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="PointsBackground2">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>550</y>
       <width>190</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 64);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>PointsBackground2</zorder>
    <zorder>TimeBackground2</zorder>
    <zorder>Video6</zorder>
    <zorder>HandIcon1</zorder>
    <zorder>Time2</zorder>
    <zorder>Back6</zorder>
    <zorder>Points2</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="background2.qrc"/>
  <include location="hand3.qrc"/>
  <include location="basketball2.qrc"/>
  <include location="background3.qrc"/>
  <include location="hand.qrc"/>
  <include location="basketball.qrc"/>
 </resources>
 <connections/>
</ui>
